tests/schema/tests.py

```diff
@@ -1350,6 +1350,36 @@ def test_m2m(self):
     def test_m2m_custom(self):
         self._test_m2m(CustomManyToManyField)
 
+    def test_m2m_through_indexes(self):
+        """
+        Tests for correct indexes in autocreated through model
+        """
+        class LocalAuthorWithM2M(Model):
+            name = CharField(max_length=255)
+            tags = ManyToManyField(Tag, related_name="authors")
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).